libpod/finished_64.go

```diff
@@ -1,4 +1,4 @@
-// +build !arm,!386
+// +build !arm,!386, linux
```

Comment:
Is the space before linux ok here?  Should this be 
// +build !arm,!386,linux

Reply:
fixed